legal-api/tests/unit/api/test_business_filings/test_filings.py

```diff
@@ -112,6 +112,24 @@ def test_get_one_business_filing_by_id(session, client, jwt):
     assert rv.json['filing']['business'] == ar['filing']['business']
 
 
+def test_get_one_business_filing_by_id_raw_json(session, client, jwt):
+    """Assert that the raw json originally submitted is returned."""
+    import copy
+    identifier = 'CP7654321'
+    b = factory_business(identifier)
+    filings = factory_filing(b, ANNUAL_REPORT)
+    ar = copy.deepcopy(ANNUAL_REPORT)
+    ar['filing']['header']['filingId'] = filings.id
+    ar['filing']['header']['colinIds'] = []
+
+    rv = client.get(f'/api/v1/businesses/{identifier}/filings/{filings.id}?original=true',
+                    headers=create_header(jwt, [STAFF_ROLE], identifier))
+
+    assert rv.status_code == HTTPStatus.OK
+    assert rv.json['filing']['annualReport'] == ar['filing']['annualReport']
```

Comment:
Not sure of the purpose of lines  121, 122 and 123. Here cant it be compared like ANNUAL_REPORT['filing']['annualReport']